src/main/java/reposense/model/Author.java

```diff
@@ -140,6 +140,7 @@ public PathMatcher getIgnoreGlobMatcher() {
      */
     public void appendIgnoreGlobList(List<String> ignoreGlobList) {
         validateIgnoreGlobs(ignoreGlobList);
+        this.ignoreGlobList.removeAll(ignoreGlobList);
```

Comment:
This line added does not match the name of the method. Make it a separate method like `setIgnoredGlobList` if it is truly needed.

Reply:
Using the current approach it is needed to remove the duplicates.